embeds.xml

```diff
@@ -2,19 +2,24 @@
     <Script file="Libs\LibStub\LibStub.lua"/>
     <Include file="Libs\CallbackHandler-1.0\CallbackHandler-1.0.xml"/>
     <Include file="Libs\AceAddon-3.0\AceAddon-3.0.xml"/>
-    <Include file="Libs\AceBucket-3.0\AceBucket-3.0.xml"/>
     <Include file="Libs\AceEvent-3.0\AceEvent-3.0.xml"/>
     <Include file="Libs\AceTimer-3.0\AceTimer-3.0.xml"/>
+    <Include file="Libs\AceBucket-3.0\AceBucket-3.0.xml"/>
+    <!--Include file="Libs\AceHook-3.0\AceHook-3.0.xml"/-->
     <Include file="Libs\AceDB-3.0\AceDB-3.0.xml"/>
+    <!--Include file="Libs\AceDBOptions-3.0\AceDBOptions-3.0.xml"/-->
     <Include file="Libs\AceLocale-3.0\AceLocale-3.0.xml"/>
     <Include file="Libs\AceConsole-3.0\AceConsole-3.0.xml"/>
     <Include file="Libs\AceGUI-3.0\AceGUI-3.0.xml"/>
-    <Include file="Libs\LibSharedMedia-3.0\lib.xml"/>
-    <Include file="Libs\AceGUI-3.0-SharedMediaWidgets\widget.xml"/>
     <Include file="Libs\AceConfig-3.0\AceConfig-3.0.xml"/>
     <Include file="Libs\AceComm-3.0\AceComm-3.0.xml"/>
+    <!--Include file="AceTab-3.0\AceTab-3.0.xml"/-->
     <Include file="Libs\AceSerializer-3.0\AceSerializer-3.0.xml"/>
+    <!-- Ace3 frame work end -->
+    <Include file="Libs\LibSharedMedia-3.0\lib.xml"/>
+    <Include file="Libs\AceGUI-3.0-SharedMediaWidgets\widget.xml"/>
```

Comment:
Why isn't this in the "Ace3 frame work" section?

Reply:
LibSharedMedia **and** AceGUI-3.0-SharedMediaWidgets are separate downloads from Ace3 AddOn development framework.
<https://www.wowace.com/projects/libsharedmedia-3-0>
<https://www.wowace.com/projects/ace-gui-3-0-shared-media-widgets>
vs. <https://www.wowace.com/projects/ace3>